README.md

```diff
@@ -0,0 +1,31 @@
+= pmdtester
```

Comment:
Since this is now a markdown file (that's the preferred format we use at PMD), the syntax is now a bit different:

The headlines start with `#` and `##` (for the 2nd level) instead of the equal signs.
You can see at https://github.com/djydewang/pmd-regression-tester-1 how github interprets this.